### DISCUSSION TITLE
Ποιά ημ/νία ισχύει για την εργασία 4; 20 ή 27 Μαρτίου;

### DISCUSSION THREAD

**Discussion by std133524** [sdy60/syllabus#135] [Q&A] [answered]
Καλησπέρα σας,

στο study.eap.gr, στο ημερολόγιο, η εργασία 4 οφείλεται στις **20 Μαρτίου** 

![cal_sdy60](https://github.com/sdy60/syllabus/assets/56337225/6b005c1a-cfec-40c1-95d5-d2138abc42d5)

ενώ στο github οφείλεται στις **27 Μαρτίου**. 

>  **Your assignment is due by Mar 27, 2024, 23:59** 

Δεν ήμουν παρών στην τελευταία ΟΣΣ. Αν τυχόν ειπώθηκε κάτι σχετικό, θα ήμουν ευγνώμων αν μπορούσατε να με ενημερώσετε.

Σας ευχαριστώ πολύ.

**Comment by epidrome** [comment, marked as answer]
@std133524 #134
